arci-ros/examples/cmd_vel.rs

```diff
@@ -0,0 +1,30 @@
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    use arci::{BaseVelocity, MoveBase};
+    use arci_ros::RosCmdVelMoveBase;
+    use rosrust::{init, is_ok, rate};
+
+    //	env_logger::init();
+
+    // Initialize node
+    rosrust::init("talker");
```

Comment:
Please change the node name.
And use arci_ros::init instead of rosrust::init.

```suggestion
    arci_ros::init("arci_ros_cmd_vel_example");
```

Reply:
ok. I'll challenge.